docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -94,41 +115,56 @@ oc logs deployment.apps/operator-hazelcast-platform-operator
 
 == Step 2. Start the Hazelcast Cluster
 
-After installing and running the Hazelcast Platform Operator, you can create a Hazelcast cluster. First, create the `Hazelcast` custom resource file as `hazelcast.yaml`.
+After installing and starting Hazelcast Operator, you can create and start a Hazelcast cluster as follows:
 
 [tabs]
 ====
-Open Source::
+Enterprise Edition::
 +
 --
+Hazelcast Platform (Enterprise Edition) requires a license key. 
 
-. Create the `Hazelcast` custom resource file and name it `hazelcast.yaml`.
+If you don't have a license key, you can request a trial license from the link:http://trialrequest.hazelcast.com/[Hazelcast website].
+
+. Create a Kubernetes secret to hold your license key as follows:
 +
-[source,yaml,subs="attributes+"]
+.For Kubernetes
+[source,shell]
 ----
-include::ROOT:example$/hazelcast.yaml[]
+kubectl create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
+----
++
+.For Openshift
+[source,shell]
+----
+oc create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
 ----
 
-. Apply the custom resource to start the Hazelcast cluster.
+. Create the Hazelcast custom resource file with the following content, and save it in your repository as _hazelcast-enterprise.yaml_:
```

Comment:
```suggestion
. Create the Hazelcast custom resource file with the following content, and save it in your repository as _hazelcast-enterprise.yml_:
```

Reply:
My system saves as yml not yaml - is this standard?

Do we need to:

- Update all the information to use .yml
- Reword along the lines of: Save it as a YAML file called _hazelcast-enterprise_  with a note like "The YAML file extension can be .yaml or .yml. The examples in this documentation use .yaml."

Reply:
I do not know which one is standard; both of them are functional 🙂 but I checked some documentation like k8s official, and they use YAML.

YAML Org advised to use `.yaml` also BTW 😄 
https://yaml.org/faq.html

Reply:
This change is not required - looks like this is a Windows thing, where extensions are generally three characters. Either format obviously works, but we will stick to using .yaml in the docs as it is recommended.